src/OpenConext/ProfileBundle/Resources/config/services.yml

```diff
@@ -17,6 +17,8 @@ services:
         arguments:
             - ~ # engine block entity id
 
+    OpenConext\EngineBlockApiClientBundle\Repository\InstitutionRepository:
```

Comment:
Enabling autoconfigure or autowire, I'm uncertain which one does what exactly, on the default container should mitigate having to do this.

Reply:
right again, it's autowire

Reply:
weird enough when i first tested this it worked, but after pushing it and trying it a second time it fails every time.  So i kept it in there.  Autowiring aparently doesn't work for this class, not sure why.

Reply:
No hard feelings! On the next major release of Symfony I believe we need to overhaul the container configuration. So we can fix it there and then.